utils/mil_msgs/msg/LabeledImage.msg

```diff
@@ -0,0 +1,7 @@
+# Represents the complete data from a labeled image from LabelMe
+
+sensor_msgs/Image image            # Image where objects are labeled
```

Comment:
The idea is that a new bag will be produced with just these messages, so all original images are just in these messages.